tidb-cloud/migrate-from-aurora-bulk-import.md

```diff
@@ -11,19 +11,23 @@ This document describes how to migrate data from Amazon Aurora MySQL to TiDB Clo
+1. Open the **Import** page for your target cluster.
 
-2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
+    1. Log in to the [TiDB Cloud console](https://tidbcloud.com/) and navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project.
 
-    > **Tip:**
-    >
-    > Alternatively, you can also click the name of your target cluster on the **Clusters** page and click **Import Data** in the **Import** area.
+        > **Tip:**
+        >
+        > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.
+
+    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+
+2. On the **Import** page, click **Import Data** in the upper-right corner, and then select **From S3**.
 
-3. Prepare source data according to [Learn how to create an Amazon S3 Bucket and prepare source data files](#learn-how-to-create-an-amazon-s3-bucket-and-prepare-source-data-files). You can see the advantages and disadvantages of different **Data Format** in the preparing data part.
+3. Prepare source data according to [Learn how to create an Amazon S3 Bucket and prepare source data files](#learn-how-to-create-an-amazon-s3-bucket-and-prepare-source-data-files). You can see the advantages and disadvantages of different data formats in the preparing source data files part.
 
-4. Select or fill in the **Data Format**, **Location**, **Bucket URI**, and **Role ARN** fields according to the specification of your source data. For more information about how to create the bucket policy and role for cross-account access, see [configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
+4. Select or fill in the **Data format**, **Bucket URI**, and **Role ARN** fields according to the specification of your source data. For more information about how to create the bucket policy and role for cross-account access, see [configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
 
-5. Check the cluster name and the region name in the **Target Cluster**. Click **Next**.
+5. Check the cluster name and the region name in the **Target database**. Click **Next**.
 
     TiDB Cloud starts validating whether it can access your data in the specified bucket URI. After validation, TiDB Cloud tries to scan all the files in the data source using the default file naming pattern, and returns a scan summary result on the left side of the next page. If you get the `AccessDenied` error, see [Troubleshoot Access Denied Errors during Data Import from S3](/tidb-cloud/troubleshoot-import-access-denied-error.md).
 
```

Comment:
L44:
```markdown
      For more information, see [table filter syntax](/table-filter.md#syntax).
```
<img width="908" alt="image" src="https://user-images.githubusercontent.com/60599231/210473630-29773dce-42de-4fd9-998d-1006f1391d8a.png">

Reply:
Thanks for the catch. Fixed in https://github.com/pingcap/docs/pull/11883/commits/c98e49294fa5d86ef3bf7705d3982bb3bc80dfa3